web/webplatform/nothing.zip

```diff
@@ -0,0 +1 @@
+This is not really a zip file, just a placeholder to prevent go embed from failing if no distribution was downloaded.
```

Comment:
But shouldn't it fail in this case?

Reply:
nope, sometimes web platform is not required.